content/docs/intro/concepts/stack.md

```diff
@@ -122,12 +122,30 @@ A stack can be exported to see the raw data associated with the stack.  This is
 
 > **Note:** This is a powerful capability that subverts the usual way that Pulumi manages resources and ensures immutable and repeatable infrastructure deployments.  Importing an incorrect stack specification could lead to orphaning of cloud resources or the inability to make future updates to the stack.  Use care when using the import and export capabilities.
 
+{{< chooser os "macos,windows,linux" >}}
+
+{{% choosable os "macos,linux" %}}
+
 ```bash
 $ pulumi stack export > stack.json
 
 $ pulumi stack import < stack.json
 ```
 
+{{% /choosable %}}
+
+{{% choosable os windows %}}
+
+```powershell
+> pulumi stack export > stack.json
+
+> Get-Content stack.json | pulumi stack import
+```
+
+{{% /choosable %}}
+
+{{< /chooser >}}
```

Comment:
No need to change - but an alternative would be to just move to using `—file` on all platforms (which is possibly more “natural” anyway?)

Reply:
I did debate that internally as well. For whatever reason, the idea of piping a file via `stdin` felt 'natural' in that it was the 'reverse' action of piping `stdout` to a file. I think if we're going to show `--file` then it'd be in both directions as well. I'm going to just do that, which removes the need for any choosers, which makes the page less cluttered.